.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bosh-releases/quarks-gora-release"]
```

Comment:
why do we need a submodule?

Reply:
I've done that to re-use the release-images-dev pipeline, basically it builds releases from that directory. I can create a separate pipeline if you wish ( I've tried to share as much as possible with the existing one )

Reply:
we should publish gora on bosh.io, so we get new builds when we have new releases ..
otherwise we have to bump CI in order to get new builds, which is not what we want

Reply:
oh well.. then we need a bit of more work, as we don't submit any final-release with this pipeline, neither we tag or upload the release to gh

Reply:
now it handle final release push + github rel, didn't put a trigger on master, we have to manually start it